### PR TITLE
Update Set-MpPreference.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to -CheckForSignaturesBeforeRunningScan as per the group policy description

Fixes https://github.com/MicrosoftDocs/windows-powershell-docs/issues/3251

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
CC :  @scanum

File: docset/winserver2016-ps/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Accept wildcard characters: False
 Indicates whether to check for new virus and spyware definitions before Windows Defender runs a scan.
 If you specify a value of $True, Windows Defender checks for new definitions.
 If you specify $False or don't specify a value, the scan begins with existing definitions.
-This value applies to scheduled scans and to scans that you start from the command line, but it doesn't affect scans that you start from the user interface.
+This setting applies to scheduled scans, but it has no effect on scans initiated manually from the user interface or to the ones started from the command line using "mpcmdrun -Scan"
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/defender/Set-MpPreference.md:109]:
```suggestion
This setting applies to scheduled scans, but it has no effect on scans initiated manually from the user interface or on scans started from the command line using "mpcmdrun -Scan"
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ Accept wildcard characters: False
 Indicates whether to check for new virus and spyware definitions before Windows Defender runs a scan.
 If you specify a value of `$True`, Windows Defender checks for new definitions.
 If you specify `$False` or don’t specify a value, the scan begins with existing definitions.
-This value applies to scheduled scans, but it doesn’t affect scans that you start from the user interface or to scans that you start from the command line.
+This setting applies to scheduled scans, but it has no effect on scans initiated manually from the user interface or to the ones started from the command line using "mpcmdrun -Scan"
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:117]:
```suggestion
This setting applies to scheduled scans, but it has no effect on scans initiated manually from the user interface or on scans started from the command line using "mpcmdrun -Scan"
```

File: docset/winserver2022-ps/defender/Set-MpPreference.md [hunk 3]
```diff
@@ -220,7 +220,7 @@ Accept wildcard characters: False
 Indicates whether to check for new virus and spyware definitions before Windows Defender runs a scan.
 If you specify a value of $True, Windows Defender checks for new definitions.
 If you specify $False or don't specify a value, the scan begins with existing definitions.
-This value applies to scheduled scans and to scans that you start from the command line, but it doesn't affect scans that you start from the user interface.
+This setting applies to scheduled scans, but it has no effect on scans initiated manually from the user interface or to the ones started from the command line using "mpcmdrun -Scan"
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Set-MpPreference.md:223]:
```suggestion
This setting applies to scheduled scans, but it has no effect on scans initiated manually from the user interface or on scans started from the command line using "mpcmdrun -Scan"
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2016-ps/defender/Set-MpPreference.md [hunk 4]
```diff
@@ -106,7 +106,7 @@ Accept wildcard characters: False
 Indicates whether to check for new virus and spyware definitions before Windows Defender runs a scan.
 If you specify a value of $True, Windows Defender checks for new definitions.
 If you specify $False or don't specify a value, the scan begins with existing definitions.
-This value applies to scheduled scans and to scans that you start from the command line, but it doesn't affect scans that you start from the user interface.
+This setting applies to scheduled scans, but it has no effect on scans initiated manually from the user interface or on scans started from the command line using "mpcmdrun -Scan"
```
Comment (dariomws) [inline docset/winserver2016-ps/defender/Set-MpPreference.md:109]:
```suggestion
This setting applies to scheduled scans, but it has no effect on scans initiated manually from the user interface or scans started from the command line using "mpcmdrun -Scan".
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 5]
```diff
@@ -114,7 +114,7 @@ Accept wildcard characters: False
 Indicates whether to check for new virus and spyware definitions before Windows Defender runs a scan.
 If you specify a value of `$True`, Windows Defender checks for new definitions.
 If you specify `$False` or don’t specify a value, the scan begins with existing definitions.
-This value applies to scheduled scans, but it doesn’t affect scans that you start from the user interface or to scans that you start from the command line.
+This setting applies to scheduled scans, but it has no effect on scans initiated manually from the user interface or on scans started from the command line using "mpcmdrun -Scan"
```
Comment (dariomws) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:117]:
```suggestion
This setting applies to scheduled scans, but it has no effect on scans initiated manually from the user interface or on scans started from the command line using "mpcmdrun -Scan".
```

File: docset/winserver2022-ps/defender/Set-MpPreference.md [hunk 6]
```diff
@@ -220,7 +220,7 @@ Accept wildcard characters: False
 Indicates whether to check for new virus and spyware definitions before Windows Defender runs a scan.
 If you specify a value of $True, Windows Defender checks for new definitions.
 If you specify $False or don't specify a value, the scan begins with existing definitions.
-This value applies to scheduled scans and to scans that you start from the command line, but it doesn't affect scans that you start from the user interface.
+This setting applies to scheduled scans, but it has no effect on scans initiated manually from the user interface or on scans started from the command line using "mpcmdrun -Scan"
```
Comment (dariomws) [inline docset/winserver2022-ps/defender/Set-MpPreference.md:223]:
```suggestion
This setting applies to scheduled scans, but it has no effect on scans initiated manually from the user interface or on scans started from the command line using "mpcmdrun -Scan".
```

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd hope all is well. Could you please help me merge this PR? Thanks!

**Pull request merged**